cicd/gitops/health-information-for-resources-deployment.adoc

```diff
@@ -6,6 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The environment details page displays the health status of the application resources, such as routes, synchronization status, deployment configuration and deployment history.
+The {gitops-title} *Environments* page in the *Developer* perspective of the {product-title} web console shows a list of the successful deployments of the application environments, along with links to the revision for each deployment.
+
+The *Application environments* page in the *Developer* perspective of the {product-title} web console displays the health status of the application resources, such as routes, synchronization status, deployment configuration, and deployment history.
+
+The *Application environments* pages in the *Developer* perspective of the {product-title} web console are decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. You do not have to use `kam` to generate the Application Environment manifests in order for the environments to show up in the *Developer* perspective of the {product-title} web console. You can use your own manifests, but the environments must however still be represented by namespaces. In addition, specific labels and annotations are still needed.
```

Comment:
It should be `The *Application environments* page... is decoupled from...` not `The *Application environments* pages ... are decoupled from...`. Only one page is affected by this enhancement.

Reply:
I discussed with @keith about this description, and we agreed that we don't have to be specific for which page, so we can just use 'environment pages' for the Monitoring application health section.